srv/wordpress/wp-content/plugins/acore-wp-plugin/src/components/AdminPanel/Settings.controller.php

```diff
@@ -46,21 +46,215 @@ public function loadSettings() {
         // See if the user has posted us some information
         // If they did, this hidden field will be set to 'Y'
 
-        foreach (Opts::I()->getConfs() as $key => $value) {
-            if (isset($_POST[$key])) {
-                $this->model->storeConf($key, $_POST[$key]);
+        if ($_SERVER['REQUEST_METHOD'] == 'POST') {
+            foreach (Opts::I()->getConfs() as $key => $value) {
+                if (isset($_POST[$key])) {
+                    $this->model->storeConf($key, $_POST[$key]);
+                }
             }
-        }
 
-        $this->data = $this->model->loadData(); // reload confs
-        // Put a "settings saved" message on the screen
-        ?>
-        <div class="updated"><p><strong>Option saved</strong></p></div>
-        <?php
+            $this->data = $this->model->loadData(); // reload confs
+            // Put a "settings saved" message on the screen
+            ?>
+            <div class="updated"><p><strong>Option saved</strong></p></div>
+            <?php
+        }
 
         echo $this->getView()->getSettingsRender();
     }
 
+    public function loadPvpRewards() {
+        //must check that the user has the required capability
+        if (!current_user_can('manage_options')) {
+            wp_die(__('You do not have sufficient permissions to access this page.'));
+        }
+
+        if (!is_plugin_active('mycred/mycred.php')) {
+            wp_die(__('You need mycred plugin active to use PvP Rewards.'));
+        }
+
+        // See if the user has posted us some information
+        // If they did, this hidden field will be set to 'Y'
+        $amount = 0;
+        $isWinner = 1;
+        $bracket = 0;
+        $bracketAnd = '';
+        $month = 1;
+        $year = 2010;
+        $result = [];
+        $top = 0;
+        $fixedAmount = 0;
+        $stepAmount = 0;
+        // TODO OBTAIN MYCRED FROM DATABASE.
+        $mycredTokenName = "mycred_default";

```

Comment:
what about this? this token is exactly `mycred_default`?

Reply:
It was the default mycred value (used in the sample db you sent), I updated it to use the mycred options.

Reply:
![imagen](https://user-images.githubusercontent.com/4155059/117554926-bb26e180-b028-11eb-9c7d-a5811ef30df3.png)
This was the default value